CHANGES.rst

```diff
@@ -91,6 +91,8 @@ New Features
   - Write comment lines in Table metadata during output for all basic formats,
     IPAC, and fast writers. This functionality can be disabled with
     ``comment=False``. [#3255]
```

Comment:
More substantively, while we're in here it might be worth nothing that the details of this format are provisional pending acceptance of APE<whatever>.  I definitely think we should advertise this more to get more community feedback, etc.